mis_builder/static/src/components/mis_report_widget.xml

```diff
@@ -85,21 +85,49 @@
                                     t-as="cell"
                                     t-key="cell_index"
                                     t-att="{'style': cell.style, 'title': cell.val_c}"
-                                    class="mis_builder_amount"
+                                    class="mis_builder_amount dropdown-mis-builder"
                                 >
-                                    <t t-if="cell.drilldown_arg">
+                                    <div class="btn-group dropdown">
                                         <a
                                             href="javascript:void(0)"
-                                            class="mis_builder_drilldown"
-                                            t-on-click="drilldown"
-                                            t-att-data-drilldown="JSON.stringify(cell.drilldown_arg)"
+                                            data-bs-toggle="dropdown"
+                                            t-attf-class="dropdown-toggle"
                                         >
                                             <t t-esc="cell.val_r" />
+                                            <span
+                                                t-if="cell.note_sequence"
+                                                class="footnote-mis-builder"
+                                                t-att="{'title': cell.note_text}"
+                                            >
+                                                <t t-out="cell.note_sequence" />
+                                            </span>
                                         </a>
-                                    </t>
-                                    <t t-if="!cell.drilldown_arg">
-                                        <t t-esc="cell.val_r" />
-                                    </t>
+                                        <div
+                                            class="dropdown-menu o_filter_menu"
+                                            role="menu"
+                                        >
+                                            <div
+                                                href="javascript:void(0)"
+                                                class="dropdown-item js_tag"
+                                                t-on-click="drilldown"
+                                                t-att-data-drilldown="JSON.stringify(cell.drilldown_arg)"
+                                                t-if="cell.drilldown_arg"
+                                                role="menuitem"
+                                            >
+                                                Explore
```

Comment:
Drill down instead of Explore ? not sure.

But can we keep the immediate drill down when we click on the value, and have the menu appear only when we click on the little down arrow ?